soul-plugin/soul-plugin-ratelimiter/pom.xml

```diff
@@ -56,5 +56,10 @@
             <artifactId>reactor-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.github.kstyrc</groupId>
+            <artifactId>embedded-redis</artifactId>
+            <version>0.6</version>
+        </dependency>
```

Comment:
This dependency is only for test. right? Please add `<scope>test</scope>` to your pom.xml

Reply:
ok